packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -118,8 +121,10 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
     const props = this.props;
     const strings = this._strings;
     const textLabels = this._textLabels;
-    const { theme, className, styles, alphaSliderHidden } = props;
+    const { theme, className, styles, alphaSliderHidden, useTransparencySlider } = props;
     const { color } = this.state;
+    const alphaValue = color.a;
+    const sliderValue = alphaValue == null ? 100 : this._getAlphaOrTransValue(alphaValue);
```

Comment:
Why the new default value logic here? Also this var should be called alphaSliderVa